### ISSUE TITLE
Add uninstall routine

### ISSUE THREAD

**Issue by ragulka** [skyverge/wc-plugin-framework#41]
Do we need to handle plugin uninstalls somehow? Consider the following scnario:
1. User installs plugin version 1.0
2. After a while, she uninstalls it
3. After a little while more, she installs the same plugin, version 1.2

It's important that all the install and upgrade routines are run correct each time. I'm not sure if the current implementation can lead to issues there, but let's discuss any possible shortcomings.

**Comment by unfulvio** [comment]
related #179

**Comment by tamarazuk** [comment]
I thought i had already added this in GitHub but apparently not. A nice way to handle uninstalls is to add a section/button under **System Status > Tools** so that deleting data from a plugin is a hard-to-find discrete action a merchant has to take. This goes in line with @unfulvio's note in #179 to make this a choice the merchant makes. 

IMO, this is by far a better way to handle WP plugin uninstalls than uninstall.php even when using an option. Many plugins of ours don't have an options page or only have one or 2 strategically placed options that adding one for deleting data on uninstall is a bit silly (see Nested Category Layout which adds an entire WC settings section for NCL just for this one option).

**Comment by ChaseWiseman** [comment]
>Many plugins of ours don't have an options page or only have one or 2 strategically placed options that adding one for deleting data on uninstall is a bit silly

Yep, agreed. I think your suggestion for adding a System Status tool is spot on.
